frontend/src/bundles/landing/pages/landing.tsx

```diff
@@ -0,0 +1,428 @@
+import appPreview from '~/assets/img/landing/app-preview.svg';
+import authPreview from '~/assets/img/landing/auth-preview.svg';
+import featureBg from '~/assets/img/landing/feature-bg.png';
+import logo from '~/assets/img/logo.svg';
+import {
+    Button,
+    ButtonVariant,
+    Icon,
+    Link,
+    Loader,
+    ThemeSwitcher,
+} from '~/bundles/common/components/components.js';
+import {
+    type IconName,
+    IconColor,
+} from '~/bundles/common/components/icon/enums/enums.js';
+import { ComponentSize } from '~/bundles/common/enums/component-size.enum.js';
+import { AppRoute } from '~/bundles/common/enums/enums.js';
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+import {
+    useAppDispatch,
+    useAppSelector,
+    useCallback,
+    useEffect,
+} from '~/bundles/common/hooks/hooks.js';
+import { type ValueOf } from '~/bundles/common/types/types.js';
+
+import {
+    Fade,
+    FeatureCard,
+    IntroCard,
+    TestimonialCard,
+    Zoom,
+} from '../components/components.js';
+import { FEATURES, TESTIMONIALS } from '../constants/constants.js';
+
+const Landing = (): JSX.Element => {
+    const { user, isRefreshing } = useAppSelector(({ auth }) => ({
+        user: auth.user,
+        isRefreshing: auth.isRefreshing,
+    }));
+    const dispatch = useAppDispatch();
+
+    useEffect(() => {
+        if (user) {
+            dispatch(appActions.navigate(AppRoute.OVERVIEW));
+        }
+    }, [dispatch, user]);
+
+    const signUpHandler = useCallback((): void => {
+        dispatch(appActions.navigate(AppRoute.SIGN_UP));
+    }, [dispatch]);
+
+    if (isRefreshing) {
+        return <Loader isOverflow />;
+    }
+
+    const styles = {
+        section: 'flex flex-col min-h-screen shrink-0 snap-start snap-always',
+        container: 'container mx-auto',
+        secondSection: {
+            gradient:
+                'after:absolute after:right-1/2 after:bottom-28 after:h-[39rem] after:w-[42rem] after:translate-x-1/2 after:translate-y-1/2 after:rounded-full after:bg-gradient-to-b after:from-[#E6E345] after:to-[#A1EE7D] after:opacity-20 after:blur-[100px]',
+        },
+        thirdSection: {
+            gradient:
+                'after:absolute after:bottom-0 after:left-0 after:-translate-x-1/4 after:translate-y-1/3 after:size-[22rem] after:z-[-1] after:rounded-full after:bg-gradient-to-b after:from-[#E6E345] after:to-[#A1EE7D] after:blur-[140px]',
+        },
+    };
+
+    return (
+        <main className="text-primary flex h-screen snap-y snap-mandatory flex-col gap-40 overflow-y-auto overflow-x-hidden scroll-smooth">
+            <section
+                className={getValidClassNames(styles.section, styles.container)}
+            >
+                <header className="container mx-auto my-6 flex flex-row items-center justify-between">
+                    <div>
+                        <Link to={AppRoute.ROOT}>
+                            <img src={logo} alt="Logo" />
```

Comment:
Here too
![image](https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/assets/72942456/fe834b69-e274-451a-9052-61880f373895)

Reply:
It looks really great, especially the animation) Cool work)